### PR TITLE
Get color for optimization

### PR DESCRIPTION
**Pull request by ncyellow**
small optimization getColorFor

- remove unused parameter
- replace local variable to static const variable

### CODE REVIEW

File: src/dialogs/NewFileDialog.cpp [hunk 1]
```diff
@@ -42,7 +38,7 @@ static QIcon getIconFor(const QString &str, int pos)
     QPainter pixPaint(&pixmap);
     pixPaint.setPen(Qt::NoPen);
     pixPaint.setRenderHint(QPainter::Antialiasing);
-    pixPaint.setBrush(QBrush(QBrush(getColorFor(str, pos))));
+    pixPaint.setBrush(QBrush(QBrush(getColorFor(pos))));
```
Comment (pelijah) [inline src/dialogs/NewFileDialog.cpp]: Hmm `QBrush(QBrush(...))`?

Comment (ncyellow) [reply src/dialogs/NewFileDialog.cpp]: so it was already =)

Comment (ncyellow) [reply src/dialogs/NewFileDialog.cpp]: i change only function getColorFor

Comment (pelijah) [reply src/dialogs/NewFileDialog.cpp]: Could you remove this extra call to ctor?

Comment (a1ext) [reply src/dialogs/NewFileDialog.cpp]: Lol

File: src/dialogs/NewFileDialog.cpp [hunk 2]
```diff
@@ -14,20 +14,16 @@
...
-
-    QList<QColor> Colors;
-    Colors << QColor(29, 188, 156); // Turquoise
-    Colors << QColor(52, 152, 219); // Blue
-    Colors << QColor(155, 89, 182); // Violet
-    Colors << QColor(52, 73, 94);   // Grey
-    Colors << QColor(231, 76, 60);  // Red
-    Colors << QColor(243, 156, 17); // Orange
-
+    static const QList<QColor> Colors = { QColor(29, 188, 156), // Turquoise
```
Comment (a1ext) [inline src/dialogs/NewFileDialog.cpp]: Please use Qt code style format

Comment (ncyellow) [reply src/dialogs/NewFileDialog.cpp]: you talk about name Colors instead colors?

Comment (a1ext) [reply src/dialogs/NewFileDialog.cpp]: about new line after `{` and all the following elements of a list indent

Comment (ncyellow) [reply src/dialogs/NewFileDialog.cpp]: https://github.com/radareorg/cutter/blob/46bb07d725e22c45ca75ee0dc7485a123cab1c0a/src/widgets/SegmentsWidget.cpp#L29

Comment (ncyellow) [reply src/dialogs/NewFileDialog.cpp]: just following your repository code style =)

Comment (a1ext) [reply src/dialogs/NewFileDialog.cpp]: I was spotted on the review before.
Please make it according to Qt code style :)

File: src/dialogs/NewFileDialog.cpp (hunk 1 above)
Comment (ncyellow) [reply src/dialogs/NewFileDialog.cpp]: done

Comment (pelijah) [reply src/dialogs/NewFileDialog.cpp]: Thanks.

**Comment by ITAYC0HEN** [comment]
Very nice! Thank you :)

**Pull request merged**